### DISCUSSION TITLE
Inlay hints not disabling for jsx

### DISCUSSION THREAD

**Discussion by mksonkar** [LazyVim/LazyVim#4513] [Q&A] [answered]
I've excluded inlay hints for jsx in nvim-lspconfig, with reference to the docs: https://www.lazyvim.org/plugins/lsp#nvim-lspconfig
![image](https://github.com/user-attachments/assets/fbd0ef8f-921f-474d-92f0-c8e1500e1e0e)
But I'm still seeing inlay hints in jsx files, and I've to manually disable it for each file with `<leader>uh`. Why is the option not working?
![image](https://github.com/user-attachments/assets/5ca6ac06-8062-42f5-a714-f57aee405def)

**Comment by dpetka2001** [comment]
That option takes a filetype. `.jsx` is just an extension? Open a file with `.jsx` extension and type `:set ft?` and see what prints out. That's the filetype of the file that you have to put in your configuration.

**Comment by mksonkar** [comment, marked as answer]
Thanks! running `:set ft` returned `javascriptreact`, and after updated the config it works now.
```lua
return {
  "neovim/nvim-lspconfig",
  opts = {
    ---@class PluginLspOpts
    -- Enable this to enable the builtin LSP inlay hints on Neovim >= 0.10.0
    -- Be aware that you also will need to properly configure your LSP server to
    -- provide the inlay hints.
    inlay_hints = {
      enabled = true,
      exclude = { "javascriptreact" }, -- filetypes for which you don't want to enable inlay hints
    },
  },
}

```
